features/430-passwordless/implementation.md

```diff
@@ -38,14 +38,18 @@ one of the steps after `identity:authn` step. i.e. `mfa:authn` or `mfa:setup`.
 Previously, we have two types of identities: `password` and `oauth`. Under the
 new design, `password` type should be renamed to `login_id`.
 
+Also, identity ID is removed from identity model, so that identity information
+represent a snapshot of the identity at the time of authentication.
+
 ## Session Headers
 
 The authenticator headers would be removed and replaced with the following
 headers:
 - `X-Skygear-Session-Acr`: `acr` claim, URI value.
 - `X-Skygear-Session-Amr`: `amr` claim, space separated values.
 
-Also, `X-Skygear-Session-Identity-Updated-At` header would be removed.
+Also, `X-Skygear-Session-Identity-Id` and `X-Skygear-Session-Identity-Updated-At`
```

Comment:
Rebase onto master and

- Update `370-auth-ui/gateway.md` to remove the mentioned headers
- Grep `currentIdentity` and update it as well. Not sure if we should rename it or simply remove it. If we remove it, we can remove `Identity` from `AuthResponse`.

Reply:
I don't think we should modify existing specs at the moment, otherwise we'll have no context on the changes?

I think we can keep the `Identity` object and just remove the `id` field.

Reply:
> I think we can keep the Identity object and just remove the id field.

One type or two types? `listIdentities` still return identities with IDs? Do we remove the `id` field at HTTP API level or only hide it in the SDK?

Reply:
Remove it everywhere, including all APIs, hooks, HTTP headers, and any other places. Since we're not accepting identity ID in our APIs (they're just for convience of clients), we should be able to remove them without breaking API input interfaces.